client/signup/steps/rewind-complete/index.jsx

```diff
@@ -0,0 +1,61 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+
+class RewindCompleteStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	renderStepContent = () => {
+		const { translate, wpAdminUrl } = this.props;
+
+		return (
+			<Card className="rewind-complete__card">
+				<h3 className="rewind-complete__title">{ translate( 'Your site is backing up!' ) }</h3>
+				<img className="rewind-complete__image" src="/calypso/images/upgrades/thank-you.svg" />
+				<p className="rewind-complete__description">
+					{ translate(
+						"Your site is backing up now as part of your Jetpack Premium Plan. It's doing " +
```

Comment:
The usual pattern when splitting is to try doing it when there is a period, so you could split it this `Plan.`